core/domain/topic_services.py

```diff
@@ -1730,8 +1730,16 @@ def get_chapter_counts_in_topic_summaries(
         overdue_chapters_count = 0
         total_chapter_counts = []
         published_chapter_counts = []
-        stories = [story_id_mapping[story_reference.story_id] for
-            story_reference in topic.canonical_story_references]
+        stories = []
+        for story_reference in topic.canonical_story_references:
+            if story_reference.story_id in story_id_mapping:
+                stories.append(story_id_mapping[story_reference.story_id])
+            else:
+                logging.error(
+            'Topic %s has canonical story references ' 
```

Comment:
The indentation is wrong here. This and the following lines should all be indented by 4 from the previous line. Please follow exactly how it's done elsewhere in the codebase.